### ISSUE TITLE
Playwright version 1.33.0

### ISSUE THREAD

**Issue by Anmo** [sand4rt/playwright-ct-web#9]
Hi,

I was trying the experimental playwright with your web components implementation, but I got some errors for the latest version of playwright (1.33.0)

```
Error: Package subpath './lib/mount' is not defined by "exports" in [path]/node_modules/@playwright/test/package.json
```

Was able to "run" the tests for version `1.32.1` (the run is on quotes because of #8)

**Comment by sand4rt** [comment]
Hi @Anmo, thanks for creating the issue! Will fix this later this weekend, hopefully.

**Comment by sand4rt** [comment]
just released the fix in v1.33.0
